src/editors/index.ts

```diff
@@ -134,6 +152,7 @@ export class DebuggerEditors extends TabPanel {
     editor.title.label = path;
     editor.title.caption = path;
     editor.title.closable = true;
+    editor.id = this.debuggerService.session.client.path;
```

Comment:
Wondering whether we need an `id`, or we could use `editor.title.label`?

Reply:
After the rebase it looks like it can actually be removed.

Reply:
Removed.

Reply:
:+1: